.travis.yml

```diff
@@ -2,8 +2,14 @@ language: python
 python:
   - "2.7"
   - "3.4"
+env:
+  - DJANGO_VERSION=1.7
+  - DJANGO_VERSION=1.8
+  - DJANGO_VERSION=1.9b1
 install:
   - pip install -r dev-requirements.txt
+  - pip uninstall django --yes
+  - pip install -q django==$DJANGO_VERSION --pre
```

Comment:
I don't think you need `--pre` any more per http://stackoverflow.com/questions/29871778/why-does-this-sparkpost-python-sdk-install-fail#comment52307761_29871893